### PR TITLE
OboeTester: consolidate CommDevice UI

### PR DESCRIPTION
**Pull request by philburk**
Move SCO dialog to CommDeviceView

To help debug b/282831177

### CODE REVIEW

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/CommunicationDeviceView.java [hunk 1]
```diff
@@ -122,14 +151,41 @@ private void setSpeakerPhoneOn(boolean enabled) {
...
             if (commDeviceInfo != null) {
-                text += ", CommDev=" + commDeviceInfo.getId();
+                text += ", #" + commDeviceInfo.getId();
             }
         }
-        mIsSpeakerphoneText.setText(" => " + text);
+        mSpeakerStatusView.setText(text + ",");
+
+        if (mScoState == AudioManager.SCO_AUDIO_STATE_CONNECTING) {
+            text = ":wait";
```
Comment (robertwu1) [inline apps/OboeTester/app/src/main/java/com/mobileer/oboetester/CommunicationDeviceView.java:164]: nit: wait is in a different font as CON and DISCO

File: apps/OboeTester/app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -182,42 +182,15 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toEndOf="@+id/textView" />
 
-    <LinearLayout
-        android:id="@+id/layoutBluetoothSco"
```
Comment (robertwu1) [inline apps/OboeTester/app/src/main/res/layout/activity_main.xml:186]: We may want to just test swapping between SCO and A2DP outputs. We won't be able to test this if we move this into input activities.

File: apps/OboeTester/app/src/main/java/com/mobileer/oboetester/CommunicationDeviceView.java (hunk 1 above)
Comment (philburk) [reply apps/OboeTester/app/src/main/java/com/mobileer/oboetester/CommunicationDeviceView.java:164]: Also change to DISCON

File: apps/OboeTester/app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (philburk) [reply apps/OboeTester/app/src/main/res/layout/activity_main.xml:186]: TEST OUTPUT does have the UI.
We should add it to TapToTone, etc.

**Review by robertwu1 — APPROVED** [review]

**Pull request merged**